requirements/dev.txt

```diff
@@ -8,4 +8,5 @@ coverage
 pytest-timeout
 pytest-mock
 onnx==1.13.0
-onnxruntime==1.14.1
\ No newline at end of file
+onnxruntime==1.14.1
```

Comment:
Is this package needed?

Reply:
> Is this package needed?

This is the original content that was in requirements/dev.txt. The change I made was to add pytest-csv.
![image](https://github.com/openvinotoolkit/training_extensions/assets/55648198/25eeb14b-24a4-4b48-9bb0-15cf2418e1ce)